submissions/BlueLamantine/memory-pair-game/js/index.js

```diff
@@ -0,0 +1,229 @@
+'use strict';
+
+class General {
+  constructor(startBtn) {
+    this.startButton = document.querySelector(startBtn);
+    this.cardAreas = document.querySelectorAll('.block');
+    this.gameTheme = new Audio('./audio/game_theme.mp3');
+  }
+
+  playAudio() {
+    this.gameTheme.loop = true;
+    this.gameTheme.volume = 0.2;
+    this.gameTheme.muted = false;
+    this.gameTheme.play();
+
+    document.querySelector('#musicControl').addEventListener('click', e => {
+      e.preventDefault();
+      if (this.gameTheme.muted) {
+        this.gameTheme.muted = false;
+        e.target.classList.replace('fa-volume-off', 'fa-volume-up');
```

Comment:
You can use event destructuring here

Reply:
I am not sure to put ```e.target.classList``` at const it is what you mean, but I did